src/webgl/shaders/lighting.glsl

```diff
@@ -67,6 +79,64 @@ LightResult _light(vec3 viewDirection, vec3 normal, vec3 lightVector) {
   return lr;
 }
 
+// converts the range of "value" from [min1 to max1] to [min2 to max2]
+float map(float value, float min1, float max1, float min2, float max2) {
+  return min2 + (value - min1) * (max2 - min2) / (max1 - min1);
+}
+
+vec2 mapTextureToNormal( vec3 normal ){
```

Comment:
oh should we be using your implementation of `nTOE` from the other fragment shaders instead of this? If I recall correctly, this is the older implementation that has a bug where it only uses the 0-PI range, which you fixed in your updated `nTOE`.

I think using `nTOE` would also mean we can remove the `map` function above since this is all that uses it.

Reply:
Done, i'm testing after the changes. If nToE works, which it will very likely. I'll remove the `mapTextureToNormal` one.

Reply:
done, changed the `nToE` name to `mapTextureToNormal` for better understanding name.